src/domain/orders/draft-orders/details.tsx

```diff
@@ -376,10 +376,9 @@ const DraftOrderDetails = () => {
                         </span>
                       </span>
                       <div className="flex flex-grow items-center mt-4">
-                        <ReactJson
-                          name={false}
-                          collapsed={true}
-                          src={method?.data}
+                        <JsonViewer
+                          rootName={"shipping_method"}
+                          value={method?.data}
```

Comment:
Since it was collapsed before, do we not want to keep that? If so we could add this:

```suggestion
                        <JsonViewer
                          rootName={"shipping_method"}
                          value={method?.data}
                          defaultInspectDepth={0}
```

Reply:
Nice catch, will add :)